lib/grape/validations/params_scope.rb

```diff
@@ -264,6 +264,7 @@ def new_scope(attrs, optional = false, &block)
           parent: self,
           optional: optional,
           type: type || Array,
+          group: @group,
```

Comment:
@dblock  I've added more specs and found that in case of mix of nested scopes and groups, `with` doesn't work as expected (if there is anything expected). The reason is that when a new parameter with a block is defined , a new scope is created without any link to the current group.

Reply:
Open an issue with a failed spec?

Reply:
Whether it is an issue or an undocumented feature is always hard to tell in such mature projects :)